config/config.json

```diff
@@ -1,7 +1,7 @@
 {
-    "defaultModule": "github.com/keep-network/keep-core/solidity",
+    "defaultModule": "github.com/keep-network/keep-core/solidity-v1",
     "modules": {
-        "github.com/keep-network/keep-core/solidity": {
+        "github.com/keep-network/keep-core/solidity-v1": {
             "workflow": "contracts.yml",
```

Comment:
If we decide to rename the workflow file in keep-core we would have to update it here too.

https://github.com/keep-network/keep-core/pull/2610#discussion_r713894408

Reply:
And also we'll need to update references to the module in following places:
https://github.com/keep-network/keep-ecdsa/blob/main/.github/workflows/contracts.yml#L162
https://github.com/keep-network/coverage-pools/blob/main/.github/workflows/contracts.yml#L92

Reply:
updates for keep-core: c5f058a
for ecdsa: https://github.com/keep-network/keep-ecdsa/pull/916
for cov-pool: https://github.com/keep-network/coverage-pools/pull/197